### PR TITLE
Rearranging & adding flags 

### PR DESCRIPTION
**Pull request by idalithb**
-Rearranged order, but let me know if it's an issue 
-Edited to provide additional clarity 
-Updated flags

### CODE REVIEW

File: website/pages/en/publishing/publishing-a-subgraph.mdx [hunk 1]
```diff
@@ -2,38 +2,81 @@
...
 
-Subgraphs can be published to the decentralized network directly from Subgraph Studio dashboard by clicking on the **Publish** button. Once a subgraph is published, it will be available to view in [Graph Explorer](https://thegraph.com/explorer/).
+- After publishing your subgraph to the decentralized network, you can update the metadata anytime in Subgraph Studio.
+- Once you’ve saved your changes and published the updates, they will appear in Graph Explorer.
+- It's important to note that this process will not create a new version since your deployment has not changed.
 
-- Subgraphs can be published to Arbitrum One. [Learn more about The Graph Network on Arbitrum](/arbitrum/arbitrum-faq).
+## Publishing from the CLI
+
+As of version 0.73.0, you can also publish your subgraph with the `graph-cli`.
```
Comment (saihaj) [inline website/pages/en/publishing/publishing-a-subgraph.mdx:36]:
```suggestion
As of version 0.73.0, you can also publish your subgraph with the `graph-cli`.
```
```suggestion
As of version 0.73.0, you can also publish your subgraph with the [`graph-cli`](https://github.com/graphprotocol/graph-tooling/tree/main/packages/cli).
```

File: website/pages/en/publishing/publishing-a-subgraph.mdx [hunk 2]
```diff
@@ -2,38 +2,81 @@
...
+
+FLAGS
+  -h, --help            Show CLI help.
+  -i, --ipfs=<value>    [default: https://api.thegraph.com/ipfs/api/v0] Upload build results to an IPFS node.
+  --ipfs-hash=<value>   IPFS hash of the subgraph manifest to deploy.
+  --webapp-url=<value>  [default: https://cli.thegraph.com/publish] URL of the web UI you want to use to deploy.
+  --subgraph-id=<value>     ID of the subgraph to update.
+  --protocol-name=<value>   Name of the protocol for the subgraph update.
+
+```
```
Comment (saihaj) [inline website/pages/en/publishing/publishing-a-subgraph.mdx:60]:
```suggestion
```
USAGE
  $ graph publish [SUBGRAPH-MANIFEST] [-h] [--protocol-network arbitrum-one|arbitrum-sepolia --subgraph-id <value>] [-i <value>] [--ipfs-hash <value>] [--webapp-url
    <value>]

FLAGS
  -h, --help                   Show CLI help.
  -i, --ipfs=<value>           [default: https://api.thegraph.com/ipfs/api/v0] Upload build results to an IPFS node.
  --ipfs-hash=<value>          IPFS hash of the subgraph manifest to deploy.
  --protocol-network=<option>  [default: arbitrum-one] The network to use for the subgraph deployment.
                               <options: arbitrum-one|arbitrum-sepolia>
  --subgraph-id=<value>        Subgraph ID to publish to.
  --webapp-url=<value> 

```
```

File: website/pages/en/publishing/publishing-a-subgraph.mdx [hunk 3]
```diff
@@ -2,38 +2,81 @@
...
+  $ graph publish [SUBGRAPH-MANIFEST] [-h] [-i <value>] [--ipfs-hash <value>] [--webapp-url <value>] [--subgraph-id <value>] [--protocol-name <value>]
+
+FLAGS
+  -h, --help            Show CLI help.
+  -i, --ipfs=<value>    [default: https://api.thegraph.com/ipfs/api/v0] Upload build results to an IPFS node.
+  --ipfs-hash=<value>   IPFS hash of the subgraph manifest to deploy.
+  --webapp-url=<value>  [default: https://cli.thegraph.com/publish] URL of the web UI you want to use to deploy.
+  --subgraph-id=<value>     ID of the subgraph to update.
+  --protocol-name=<value>   Name of the protocol for the subgraph update.
+
```
Comment (saihaj) [inline website/pages/en/publishing/publishing-a-subgraph.mdx:59]:
```suggestion
USAGE
  $ graph publish [SUBGRAPH-MANIFEST] [-h] [--protocol-network arbitrum-one|arbitrum-sepolia --subgraph-id <value>] [-i <value>] [--ipfs-hash <value>] [--webapp-url
    <value>]

FLAGS
  -h, --help                   Show CLI help.
  -i, --ipfs=<value>           [default: https://api.thegraph.com/ipfs/api/v0] Upload build results to an IPFS node.
  --ipfs-hash=<value>          IPFS hash of the subgraph manifest to deploy.
  --protocol-network=<option>  [default: arbitrum-one] The network to use for the subgraph deployment.
                               <options: arbitrum-one|arbitrum-sepolia>
  --subgraph-id=<value>        Subgraph ID to publish to.
  --webapp-url=<value>         [default: https://cli.thegraph.com/publish] URL of the web UI you want to use to deploy.

```

File: website/pages/en/publishing/publishing-a-subgraph.mdx [hunk 4]
```diff
@@ -2,38 +2,81 @@
...
+- Once you’ve saved your changes and published the updates, they will appear in Graph Explorer.
+- It's important to note that this process will not create a new version since your deployment has not changed.
 
-- Subgraphs can be published to Arbitrum One. [Learn more about The Graph Network on Arbitrum](/arbitrum/arbitrum-faq).
+## Publishing from the CLI
+
+As of version 0.73.0, you can also publish your subgraph with the `graph-cli`.
+
+1. Open the `graph-cli`.
+2. Use the following commands: `graph publish` then `graph build && graph codegen`
```
Comment (saihaj) [inline website/pages/en/publishing/publishing-a-subgraph.mdx:39]: first you want to codegen -> build and finally publish the built artifacts.


```suggestion
2. Use the following commands: `graph codegen && graph build` then `graph publish`.
```

File: website/pages/en/publishing/publishing-a-subgraph.mdx [hunk 5]
```diff
@@ -46,27 +89,4 @@ Alternatively, you can add GRT signal to a published subgraph from Graph Explore
...
-USAGE
-  $ graph publish [SUBGRAPH-MANIFEST] [-h] [-i <value>] [--ipfs-hash <value>] [--webapp-url <value>]
-
-FLAGS
-  -h, --help            Show CLI help.
-  -i, --ipfs=<value>    [default: https://api.thegraph.com/ipfs/api/v0] Upload build results to an IPFS node.
-  --ipfs-hash=<value>   IPFS hash of the subgraph manifest to deploy.
-  --webapp-url=<value>  [default: https://cli.thegraph.com/publish] URL of the web UI you want to use to deploy.
-```
+Learn more about [Curating](https://thegraph.com/docs/en/network/curating/) on The Graph Network.
```
Comment (benface) [inline website/pages/en/publishing/publishing-a-subgraph.mdx:92]:
```suggestion
Learn more about [Curating](/network/curating/) on The Graph Network.
```

**Review by saihaj — APPROVED** [review]
looks great! thank you

**Review by p-diogo — APPROVED** [review]
I love these changes! It is a lot clearer, and the extra links to other pages for more information are always appreciated.

**Pull request merged**